src/hw/hw_mngr.cpp

```diff
@@ -357,3 +407,166 @@ void HWManager::setFanSpeedDelayed()
         setFanSpeed();
     });
 }
+
+void HWManager::detectFansDelayed()
+{
+    static boost::asio::deadline_timer tmr(io);
+
+    //  if we has expired attempts, then stop fan detection and work in normal
+    //  state
+    if (++numErrorAttempts > maxErrorAttempts)
+    {
+        log<level::ERR>("FAN Detection",
+                        entry("WHAT=reached maxErrorAttempts in state=%d",
+                              int(detectFansState)));
```

Comment:
This will give a message like:
```
Jul 14 15:04:23 yadro-hw-manager[441]: FAN Detection
```
The `WHAT` filed will be available only in `verbose` output.

I guess it should be:
```cpp
log<level::ERR>("Reached maxErrorAttempts", entry("STATE=%d", int(detectFansState));
```

Reply:
Fixed.